src/dmd/traits.d

```diff
@@ -651,6 +652,22 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
         }
         return False();
     }
+    if (e.ident == Id.isCopyable)
+    {
+        if (dim != 1)
+            return dimError(1);
+
+        auto o = (*e.args)[0];
+        auto t = isType(o);
+        if (!t)
+        {
+            e.error("type expected as second argument of __traits `%s` instead of `%s`",
+                    e.ident.toChars(), o.toChars());
+            return new ErrorExp();
+        }
+
```

Comment:
Get the base type in case it's an enum.
```d
struct S { @disable this(this); }
enum E : S { a = S(), }
pragma(msg, __traits(isCopyable, E)); // true
```
```suggestion

        t = t.toBasetype();
```